src/Entity/ApiDoc.php

```diff
@@ -122,16 +139,41 @@ public function setCreatedTime(int $timestamp) : ApiDocInterface {
   /**
    * {@inheritdoc}
    */
-  public function isPublished() : bool {
-    return (bool) $this->getEntityKey('status');
+  public function preSave(EntityStorageInterface $storage) {
+    parent::preSave($storage);
+
+    // If no revision author has been set explicitly, make the current user
+    // the revision author.
+    if (!$this->getRevisionUser()) {
+      $this->setRevisionUserId(\Drupal::currentUser()->id());
```

Comment:
I believe you said on an earlier comment this is how node entities work.  I would expect if there is not an author the author should be anonymous user, but wonder this is for new entities that are not yes saved.

Reply:
I think we should look at this in a follow-up. @arlina-espinoza Do you remember why this had to be added?

Reply:
Added issue #18 for this